modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/deployer/ObjectDefinitionDeployerImpl.java

```diff
@@ -608,7 +608,7 @@ private List<ServiceRegistration<?>> _registerExceptionMappers(
 						"osgi.jaxrs.application.select",
 						StringBundler.concat(
 							"(|(liferay.objects.exception.mapper=true)",
-							"(osgi.jaxrs.name= ", jaxRsApplicationName, "))")
+							"(osgi.jaxrs.name=", jaxRsApplicationName, "))")
```

Comment:
:open_mouth:

Reply:
yes, in the end it was a space matter, @4lejandrito has discovered it. Nonetheless, I was able to learn about different ExceptionMappers along the way, so I don't consider it a waste of time :D

Reply:
:woman_astronaut: